src/main/java/meeju/GoodByeMessage.java

```diff
@@ -0,0 +1,18 @@
+package meeju;
+
+/**
+ * This class handles the termination of program. Once invoked, cleanup tasks are performed
+ * and the goodbye message is printed out.
+ */
+public class GoodByeMessage {
+
+    /**
+     * Prints out goodbye message before application terminates
+     *
+     */
+
+    public static String goodByeMessage() {
```

Comment:
Should the method name be `getGoodByeMessage()` to make it sound more like an action?